src/megaclient.cpp

```diff
@@ -1853,7 +1864,7 @@ void MegaClient::exec()
                                 pendingcs = NULL;
                                 csretrying = false;
 
-                                reqs.servererror(API_ESSL, this);
+                                reqs.servererror("-23", this);
```

Comment:
we can use std::to_string(API_ESSL) here, to avoid hard-coding magic constants and make it clear which error we are producing.   Performance is not important for this case, if that was the concern.  thanks